### ISSUE TITLE
`SpaceAfterVariableName` not linting?

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#521]
Use the docs as an example, I'm not getting lint from `SpaceAfterVariableName`.

```scss
// sass/test.scss

$my-var : 0;
$my-other-variable:0;

.widget {
   color: black;      
}       
```

```bash
$ scss-lint --version
scss-lint 0.40.1
$ sass --version
Sass 3.4.14 (Selective Steve)
$ scss-lint sass/test.scss
sass/test.scss:6 [W] Indentation: Line should be indented 4 spaces, but was indented 3 spaces
sass/test.scss:6 [W] TrailingWhitespace: Line contains trailing whitespace
sass/test.scss:6 [W] ColorKeyword: Color `black` should be written in hexadecimal form as `#000000`
sass/test.scss:6 [W] ColorVariable: Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.
sass/test.scss:7 [W] TrailingWhitespace: Line contains trailing whitespace
```

**Comment by sds** [comment]
Ah, that's unfortunate. Looks like the original implementation in bdc4521 nested it under the incorrect module.

I've fixed this in 5c29141. Thanks for the report!
